### PR TITLE
Generate both x86 and x64 Windows installers.

### LINKED ISSUE #12587: Switch to 64-bit executables?
Modern games and operating systems tend to be 64-bit only these days. How about we do that switch as well? My hope is that it will address issues like https://github.com/OpenRA/OpenRA/issues/12494 as we gain more access to memory and maybe it will improve overall performance. Is that too naive? My technical knowledge in that regard is limited.

Porting should be easy. For the managed code just flip some switches and for the system calls we only need to update the bundled native Windows/Mac dependencies to their 64 bit counterparts and for Linux just update package architecture metadata.

I haven't checked how many players are still running 32-bit operating systems. Is that possible from our player survey? Hopefully no one will be excluded. My assumption or rather wishful thinking would be that most of them could upgrade to a 64-bit OS for their own benefit and 32-bit only era hardware will also make playing OpenRA especially the upcoming TS/RA2 mods not much fun anyway.

### LINKED ISSUE #16534: Project shows as broken in MonoDevelop and won't compile
Screenshot from 2019-05-12 13-04-01

### PR DESCRIPTION
**Pull request by pchote**
Investigating #16534 in more detail made me realize that our legacy workaround of building in x86 for Windows and then running the same binary as 64 bit with Mono is no longer viable.

This PR takes our toolchain another step closer to normality by changing our default platform to Any CPU (which defaults to 64 bit), with a custom configuration that produces a 32bit-preferred assembly for Windows. It then takes the next logical step of producing both 32 and 64 bit Windows installers, which avoids the hassle of trying to dynamically resolve/install our native deps.

Fixes #16534.
Fixes debugging not working in vs:mac.
Fixes #12587.
Supersedes #14675.

Test installers available at https://github.com/pchote/OpenRA/releases/tag/pkgtest-20190522

### CODE REVIEW

**Review by teinarss — DISMISSED** [review]
Tested both windows installers and the solution in VS19.

**Review by MustaphaTR — DISMISSED** [review]
Both test installers and manual building both with VS and make.cmd works fine.

File: OpenRA.Game/OpenRA.Game.csproj [hunk 1]
```diff
@@ -27,6 +26,9 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <Optimize>false</Optimize>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)' == 'Release-x86'">
```
Comment (reaperrr) [inline OpenRA.Game/OpenRA.Game.csproj]: Not sure how important, but the spacing is different from the Red Alert entry below.

Comment (pchote) [reply OpenRA.Game/OpenRA.Game.csproj]: Added a new commit to fix the whitespace in the other entries.

**Review by abcdefg30 — APPROVED** [review]

**Review by ghost — APPROVED** [review]
I was able to build and run the game via MD, and the example RA AppImage works. Building and running from the command line works, too.

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/5c095c5257e7369cc6f7ad934aae5184ae428590...70a2a0ba82c71dd95410bff5017f1cfc66032a5e)

**Pull request merged**